runtime/compiler/build/files/common.mk

```diff
@@ -257,6 +257,8 @@ JIT_PRODUCT_SOURCE_FILES+=\
     compiler/codegen/J9GCStackAtlas.cpp \
     compiler/codegen/J9Instruction.cpp \
     compiler/codegen/J9Snippet.cpp \
+    compiler/codegen/J9WatchedInstanceFieldSnippet.cpp \
+    compiler/codegen/J9WatchedStaticFieldSnippet.cpp \
```

Comment:
Can we put these in alphabetical order? Similarly for CMakeLists.txt.

Reply:
d9baf57